src/test/settings.test.ts

```diff
@@ -30,11 +30,10 @@ test('finbot > sets the appveyor token for a user', (t) => {
 
   respondStub.callsArgWith(1, response);
 
-  const robotBrain = new MockRobotBrain();
-  robot.brain = robotBrain;
-  const brainSetSpy = sinon.spy(robotBrain, 'set');
+  const secureBrain = new MockSecureBrain();

```

Comment:
I guess if I ripped out each test case into a single file I could do that. They could then look like BDD/jasmine style tests. I'll have a hack at it later and see.

But to be honest, if like to merge this first and bike shed some changes later.